### PR TITLE
Changes to incorporate change in elide async api models

### PR DESCRIPTION
**Pull request by ayeswarya**
Resolves #

## Description

Changes to incorporate new design for elide async models
https://docs.google.com/document/d/1-m05W5PJEZUOYkYq8eXIllEyfWR8ovu6K1GdRGiX6SE/edit?ts=5f7c8455
## Proposed Changes

-

## License

I confirm that this contribution is made under an MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: packages/data/addon/adapters/facts/interface.ts [hunk 1]
```diff
@@ -86,13 +86,15 @@ export enum QueryStatus {
...
-  EMBEDDED = 'EMBEDDED'
+  CSV = 'CSV',
+  JSON = 'JSON'
 }
 
 export interface AsyncQuery {
   requestId: string;
   request: RequestV1 | RequestV2;
   status: QueryStatus;
+  asyncAfterSeconds: number;
```
Comment (jkusa) [inline packages/data/addon/adapters/facts/interface.ts:97]:
```suggestion
```
Don't think we need this at the moment

File: packages/data/addon/gql/schema.js [hunk 2]
```diff
@@ -253,8 +290,14 @@ const schema = gql`
     FAILURE
   }
 
+  enum QueryResultType {
```
Comment (jkusa) [inline packages/data/addon/gql/schema.js:293]:
```suggestion
  enum TableExportResultType {
```

File: packages/data/addon/mirage/handlers/graphql.js [hunk 3]
```diff
@@ -477,7 +477,6 @@ const OPTIONS = {
       if (op === 'UPSERT' && existingQueries.length === 0) {
         const node = asyncQueries.insert({
           id: data.id,
-          asyncAfterSeconds: 10,
```
Comment (mattgraham1995) [inline packages/data/addon/mirage/handlers/graphql.js:480]: why is this being removed? Should it just be changed to `asyncAfterSeconds: data.asyncAfterSeconds,`?

**Review by mattgraham1995 — APPROVED** [review]

File: packages/data/addon/mirage/handlers/graphql.js (hunk 3 above)
Comment (ayeswarya) [reply packages/data/addon/mirage/handlers/graphql.js:480]: This was a mistake. I have added it back. thank you

**Review by jkusa — APPROVED** [review]

**Pull request merged**